src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private int size;
+    private int currentCapacity;
+    private int currentLoadFactor;
+    private Node<K, V>[] storage;
+
+    public MyHashMap() {
+        currentCapacity = DEFAULT_INITIAL_CAPACITY;
+        currentLoadFactor = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+        storage = (Node<K, V>[]) new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
That's a debatable moment with this cast, don't change it.
But I'd like to know mentor's point of view after aprrovement.

Reply:
ok